tools/fv_nudge.F90

```diff
@@ -800,7 +800,8 @@ subroutine ps_nudging(dt, factor, factor_nwp, npz, ak, bk, ps_obs, mask, tm, ps,
            pt0 = tm(i,j)/(pk0(km+1)-pk0(km))*(kappa*(pn0(km+1)-pn0(km)))
            pst = pk0(km+1) + (gz0(i,j)-phis(i,j))/(cp_air*pt0)
       endif
-666   ps_dt(i,j) = pst**(1./kappa) - ps(i,j)
+      ps_dt(i,j) = pst**(1./kappa) - ps(i,j)
```

Comment:
In the original code, 'goto 666' will execute the ps_dt(i,j) assignment. The revised code will not.